Content.Server/_DV/CosmicCult/Components/CosmicFinaleComponent.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+
+namespace Content.Server._DV.CosmicCult.Components;
+
+[RegisterComponent, AutoGenerateComponentPause]
+public sealed partial class CosmicFinaleComponent : Component
+{
+    [DataField]
+    public FinaleState CurrentState = FinaleState.Unavailable;
+
+    [DataField]
+    public bool FinaleDelayStarted = false;
+
+    [DataField]
+    public bool FinaleActive = false;
+
+    [DataField]
+    public bool Occupied = false;
+
+    [AutoPausedField] public TimeSpan FinaleTimer = default!;
+    [AutoPausedField] public TimeSpan BufferTimer = default!;
+    [AutoPausedField] public TimeSpan CultistsCheckTimer = default!;
```

Comment:
Are these not also supposed to be `DataField`?

```suggestion
    [DataField, AutoPausedField] public TimeSpan FinaleTimer = default!;
    [DataField, AutoPausedField] public TimeSpan BufferTimer = default!;
    [DataField, AutoPausedField] public TimeSpan CultistsCheckTimer = default!;
```